### DISCUSSION TITLE
q9216 camera. sending commands via TCIP/IP or UDP

### DISCUSSION THREAD

**Discussion by kb0sik** [AxisCommunications/discussions#245] [VAPIX]
I have a situation were the client wants to be able to "Hide" certain cameras during certain procedures. It is a surgery room. We have 6 unique rooms and they are controlled by a small IP push button wall plate that can only accept commands via TCP/IP or UDP.  We came up with the idea of creating a a rule about privacy mask then activate and un activate at the push of a button. Does anyone have any ideas?

**Comment by vivekatoffice** [comment]
Related link:
[287](https://github.com/orgs/AxisCommunications/discussions/287)
